src/components/Rules/styled.js

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+
+export const Rule = styled.div`
+  z-index: 10;
+  font-family: Arial, Helvetica, sans-serif;
+  position: fixed;
+  top: 5%;
+  bottom: 5%;
+  right: 5%;
+  left: 5%;
+  margin: auto;
+  max-width: 80ch;
+  border-radius: 25px;
+  border: 2px outset var(--red);
+  background: var(--red);
+  color: #fff;
+  box-shadow: 2px 2px 4px rgba(0,0,0,0.4);
+  text-shadow: 2px 2px 4px rgba(0,0,0,0.4);
+  padding: 20px;
+  cursor: default;
+  & h1 {
+    font-family: 'Luckiest Guy', cursive;
```

Comment:
Once you change and style everything in paragraphs, I don't think we will need this.